huxley/api/permissions.py

```diff
@@ -91,6 +91,23 @@ def has_permission(self, request, view):
         return False
 
 
+class DelegateListPermission(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        user = request.user
+        if user.is_superuser:
+            return True
+
+        method = request.method
+        if method == 'POST' or method == 'PUT' or method == 'PATCH':
+            return True
```

Comment:
We should probably not let anybody update whatever delegates they want. Checking for it is a little less than straightforward though. Basically, we want to check that there is no delegate for which the `school_id` is not equal to the advisor's `school_id`. Something like this might work:
```python
school_id = request.user.is_authenticated() and request.user.school_id
if not school_id:
    return False
# Get delegate_ids via list comprehension...
return not Delegate.objects.filter(id__in=delegate_ids).exclude(school_id=school_id).exists()
```